experimental/packages/opentelemetry-sdk-node/src/types.ts

```diff
@@ -14,27 +14,28 @@
  * limitations under the License.
  */
 
-import { SpanAttributes, TextMapPropagator, Sampler } from '@opentelemetry/api';
 import type { ContextManager } from '@opentelemetry/api';
+import { Sampler, Attributes, TextMapPropagator } from '@opentelemetry/api';
 import { InstrumentationOption } from '@opentelemetry/instrumentation';
+import { Resource, ResourceAttributes } from '@opentelemetry/resources';
 import { MetricExporter, Processor } from '@opentelemetry/sdk-metrics-base';
-import { Resource } from '@opentelemetry/resources';
 import {
   SpanExporter,
-  SpanProcessor,
   SpanLimits,
+  SpanProcessor,
 } from '@opentelemetry/sdk-trace-base';
 
 export interface NodeSDKConfiguration {
   autoDetectResources: boolean;
   contextManager: ContextManager;
-  defaultAttributes: SpanAttributes;
+  defaultAttributes: Attributes;
   textMapPropagator: TextMapPropagator;
   metricProcessor: Processor;
   metricExporter: MetricExporter;
   metricInterval: number;
   instrumentations: InstrumentationOption[];
   resource: Resource;
+  resourceAttributes?: ResourceAttributes;
```

Comment:
should we retype this to force the definition of service name here ?

Reply:
Do you mean like service name as a required prop? I don't think so since it can be defined via the OTEL_SERVICE_NAME environment variable if I'm not mistaken

Reply:
yeah right